en/docs/developer-portal/applications/new-application.md

```diff
@@ -4,10 +4,10 @@
 
 <div class="content"> 
```

Comment:
@divyaamunugama - These files in the `developer-portal` older can be removed from the repo yeah?

Reply:
yes, these can be removed